pkgs/applications/science/logic/potassco/clingcon.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchFromGitHub, cmake, bison, re2c }:
+
+stdenv.mkDerivation rec {
+  pname = "clingcon";
+  version = "3.3.0";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "potassco";
+    repo = "${pname}";
+    rev = "d36fd87c8937b0ae2b39593aab26f3eb717749d3";
+    fetchSubmodules = true;
+    sha256 = "1q7517h10jfvjdk2czq8d6y57r8kr1j1jj2k2ip2qxkpyfigk4rs";
+   };
+
+  postPatch = ''
+    # deal with clingcon through git submodules recursively importing
+    # an outdated version of libpotassco which uses deprecated <xlocale.h> header in .cpp files
```

Comment:
Please put the comment _outside_ `postPatch`.

Reply:
A slightly more detailed explanation: changing the comment should not (per se) change the output path (which depends on the hash of all the shell snippets).